src/components/members/MembersTable.js

```diff
@@ -271,7 +325,7 @@ const MembersTable = ({ members: rows, tableTitle, id: tableId, empty }) => {
         <TablePagination
           rowsPerPageOptions={ROWS_PER_PAGE_OPTIONS}
           component="div"
-          count={rows.size}
+          count={filteredRows.size}
           rowsPerPage={rowsPerPage}
           page={page}
           onChangePage={handleChangePage}
```

Comment:
Looks like a lot like MembersTable. Is it possible to factor out similar components?

Reply:
I will open an issue to factoring out all the tables